diagnostic.md

```diff
@@ -27,7 +27,11 @@ on the body element. Use that variable to increase font-size of `h1`
 elements by 30%.
 
 ```scss
-// your response here
+$h1-font-size:($base-font-size * 1.3);
+
+h1 {
+  font-size: $h1-font-size:;
+}
```

Comment:
This is good practice, but possibly overkill in this specific case (since presumably `$h1-font-size` will only be applied to h1 elements).